ros/api/v1/suggested_instance_types.py

```diff
@@ -71,8 +74,12 @@ def get(self):
         for row in query_result:
             # FIXME: As of now we only support AWS cloud, so statically adding it to the dict. Fix this code block
             #  upon supporting multiple clouds.
-            record = {'instance_type': row.top_candidate, 'cloud_provider': 'AWS', 'system_count': row.system_count,
-                      'description': instance_types_desc_dict()[row.top_candidate]}
+            record = {'instance_type': row.top_candidate, 'cloud_provider': 'AWS', 'system_count': row.system_count}
+            try:
+                record['description'] = instance_types_desc_dict()[row.top_candidate]
+            except KeyError:
+                LOG.info(f"Unable to get the description for {row.top_candidate}! "
```

Comment:
Instead of raising an exception, we can send nothing(`None`) so that it can be easily handled on the UI side. Like OS and other fields, we can show `NA` on the UI when values are not available.